closure/goog/math/long.js

```diff
@@ -310,7 +310,7 @@ goog.math.Long.prototype.toString = function(opt_radix) {
   var result = '';
   while (true) {
     var remDiv = rem.div(radixToPower);
-    var intval = rem.subtract(remDiv.multiply(radixToPower)).toInt();
+    var intval = rem.subtract(remDiv.multiply(radixToPower)).toInt() >>> 0; // wraps around for base 36 (dcode)
```

Comment:
Can you move the comment above the intval declaration and provide a brief explanation of why the 0 bit shift is necessary? You can be concise and reference the github issue.